integrations/apollo/src/index.ts

```diff
@@ -23,8 +22,7 @@ export type NhostApolloClientOptions = {
   fetchPolicy?: WatchQueryFetchPolicy
   connectToDevTools?: boolean
   cache?: InMemoryCache
-  onError?: RequestHandler
```

Comment:
We generally try to avoid breaking changes. Instead of dropping it right away, we could still keep `onError` and mark it as deprecated on the interface. We should still add it to the merged links for now. The same goes for `link`.

Reply:
Sounds good to me I can make the change.